proxy/api/src/http/error.rs

```diff
@@ -163,6 +163,27 @@ pub async fn recover(err: Rejection) -> Result<impl Reply, Infallible> {
                         )
                     },
                 },
+                error::Error::Keystore(keystore_err) => {
+                    if keystore_err.is_invalid_password() {
```

Comment:
Should we keep with the glossary on function names as well?
```suggestion
                    if keystore_err.is_invalid_passphrase() {
```